bower.json

```diff
@@ -5,20 +5,18 @@
   "author": "Joe Allan Muharsky <jmuharsky@gmail.com>",
   "dependencies": {},
   "devDependencies": {
-    "angular": "~1.4.7",
-    "angular-mocks": "~1.4.7",
+    "angular": "~1.4.8",
```

Comment:
That's fair, although some of the dependency updates were recommended by the angular-markdown inclusion.  Will keep them separate going forward.